pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -490,6 +494,200 @@ func TestNewFrameworkErrors(t *testing.T) {
+				PostBind:   config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+			},
+		},
+		{
+			name: "disable MultiPoint plugin at some extension points",
+			plugins: &config.Plugins{
+				MultiPoint: config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: testPlugin},
+					},
+				},
+				PreScore: config.PluginSet{
+					Disabled: []config.Plugin{
+						{Name: testPlugin},
+					},
+				},
+				Score: config.PluginSet{
+					Disabled: []config.Plugin{
+						{Name: testPlugin},
+					},
+				},
+			},
+			wantPlugins: &config.Plugins{
+				QueueSort:  config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PreFilter:  config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				Filter:     config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PostFilter: config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				Reserve:    config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				Permit:     config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PreBind:    config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				Bind:       config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PostBind:   config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+			},
+		},
+		{
+			name: "Multiple MultiPoint plugins",
+			plugins: &config.Plugins{
+				MultiPoint: config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: testPlugin},
+						{Name: scorePlugin1},
+					},
+				},
+			},
+			wantPlugins: &config.Plugins{
+				QueueSort:  config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PreFilter:  config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				Filter:     config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PostFilter: config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PreScore: config.PluginSet{Enabled: []config.Plugin{
+					{Name: testPlugin},
+					{Name: scorePlugin1},
+				}},
+				Score: config.PluginSet{Enabled: []config.Plugin{
+					{Name: testPlugin, Weight: 1},
+					{Name: scorePlugin1, Weight: 1},
+				}},
+				Reserve:  config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				Permit:   config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PreBind:  config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				Bind:     config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+				PostBind: config.PluginSet{Enabled: []config.Plugin{{Name: testPlugin}}},
+			},
+		},
+		{
+			name: "Reorder MultiPoint plugins",
+			plugins: &config.Plugins{
+				MultiPoint: config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: testPlugin},
+						{Name: scorePlugin1},
+					},
+				},
+				PreScore: config.PluginSet{
+					Disabled: []config.Plugin{
+						{Name: "*"},
+					},
+				},
+				Score: config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: scorePlugin1},
```

Comment:
what happens if you have a third multi point plugin that is not included here?

Reply:
The specific plugins take precedence over multipoint (the hierarchy is `Specific > MultiPoint > Default`). Added a 3rd plugin to show this in this example, but this is also kind of covered by the various `append` tests for `mergePlugins` (https://github.com/kubernetes/kubernetes/blob/bf2e6670f6c9a447b6c10170bf9bd5c47e4c0bf4/pkg/scheduler/apis/config/v1beta3/default_plugins_test.go#L373 for MultiPoint, but there are other cases for <v1beta3 defaults I believe)